internal/service/infra_mgmt/api_services_resource.go

```diff
@@ -79,12 +82,26 @@ func (r *ServicesResource) Create(ctx context.Context, req resource.CreateReques
 	res := apiRes.GetResult()
 	data.Flatten(ctx, &res, &resp.Diagnostics)
 
+	if data.DesiredState.ValueString() == "start" {
+		err = r.waitServiceStarted(ctx, data.Name.ValueString(), 20*time.Minute)
```

Comment:
Shouldn't you be using the default value only if the Timeout isn't set in the TF file?

Reply:
Yes. Fixed.